xapian-letor/api/Makefile.mk

```diff
@@ -1,9 +1,11 @@
 noinst_HEADERS +=\
-	api/featurevector_internal.h
+	api/featurevector_internal.h\
+	api/featurelist_internal.h
```

Comment:
you mean featurelist_internal.h should be placed in xapian-letor/include/xapian-letor folder or should I just swap their positions ??